lib/nudge_web/controllers/site_controller.ex

```diff
@@ -1,25 +1,23 @@
 defmodule NudgeWeb.SiteController do
   use NudgeWeb, :controller
+  alias Nudge.Accounts
+  alias Nudge.Accounts.Site
 
   def index(conn, _params) do
-    site = Nudge.Accounts.list_sites()
-    current_user_id = Nudge.Accounts.get_site!(site.id)
-
-    user_site =
-      Site
-      |> Nudge.Accounts.list_user_sites(current_user_id)
-      |> Repo.all()
+    user_id = get_session(conn, :user_id)
+    current_user_id = Accounts.get_user(user_id)
```

Comment:
This is a redundant db query.
We don't have to query user here again. The get_current_user plug already queries this and puts `current_user` into assigns. Just take it directly from `conn.assigns.current_user.id`